### PR TITLE
Eslit add `Next`

### PR DESCRIPTION
**Pull request by RedFox0x20**
Resolving #103 

- Added `Next` ESLint plugin.
- Fixed new linting errors.
    - Alt text added to the `Powered by vercel` image in the `Footer`.
    - Empty `alt` text added to the `Card` component, it may be desirable to expose this as a property of the component.

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ export default function Footer() {
       <p>
         <a href="https://vercel.com?utm_source=margaritahumanitarian&utm_campaign=oss">
           <Image
+            alt="Powered by vercel"
```
Comment (audreyfeldroy) [inline components/Footer.js:15]:
```suggestion
            alt="Powered by Vercel"
```

File: components/Card.js [hunk 2]
```diff
@@ -6,7 +7,7 @@ function Card({ backgroundImageSource, children }) {
     <div className="card shadow-xl image-full">
       {backgroundImageSource && (
         <figure>
-          <img src={backgroundImageSource} />
+          <Image alt="" src={backgroundImageSource} />
```
Comment (audreyfeldroy) [inline components/Card.js:10]: I like your idea of exposing the alt text as a property of the component, perhaps something like `backgroundImageAltText`?

I know it's way beyond the scope of #103, haha...I'll accept this PR and file a separate issue for it.

**Review by audreyfeldroy — APPROVED** [review]
Great work and hugely appreciated @RedFox0x20.

File: components/Card.js (hunk 2 above)
Comment (RedFox0x20) [reply components/Card.js:10]: Yep my thoughts exactly. Bit beyond my React knowledge too, currently.

**Comment by audreyfeldroy** [comment]
Thank you @RedFox0x20 🙏

**Pull request merged**